pkg/api/util/matrix/matrix_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package quota
+package matrix
```

Comment:
I feel `matrix` is a bad name because the operations only take ResourceList as input, which is not a generic data structure for matrix, and the dimension is always 1*n.